src/routes/users_routes.js

```diff
@@ -765,7 +765,7 @@ router.get('/social', ensureAuth, async (req, res) => {
 });
 
 router.post('/queuefeatured', ensureAuth, async (req, res) => {
-  const redirectTo = '/user/account?nav=patreon';
+  const redirectTo = '/user/account?nav=patreon&tab=4';
```

Comment:
Might as well go straight to the patron tab when the user is doing featured queue actions